m2c/arch_mips.py

```diff
@@ -1494,6 +1501,7 @@ def eval_fn(s: NodeState, a: InstrArgs) -> None:
         "addiu": lambda a: handle_addi(a),
         "add": lambda a: handle_add(a),
         "addu": lambda a: handle_add(a),
+        "paddub": lambda a: handle_add(a),
```

Comment:
this is dubious (paddub isn't a 128-bit add), and also shouldn't be needed -- as long as we have the paddub -> move conversion that's enough